iframe/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "iframe",
+  "version": "1.0.0",
+  "description": "",
+  "main": "login.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
We need to remove this license section.

Reply:
Your right, it makes no sense to stress over the button color right now since the end user is going to set that color.... 🤦‍♂️ .

Reply:
Okay. So yeah, still, we can finalise a default/out-of-the-box colour scheme, but later :)